generators/generator-base.js

```diff
@@ -440,8 +439,7 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
         entityUrl,
         clientFramework,
         microServiceName,
-        readOnly,
-        pageTitle
+        readOnly
```

Comment:
Keep pageTitle parameter for modules and blueprints.

Reply:
But default to `this.enableTranslation ? `${this.i18nKeyPrefix}.home.title` : this.entityClassPlural`.

Reply:
OK, keeping `pageTitle` parameter and giving default value to all parameters so that `addEntityToModule` can be called with zero parameters in `entity-client` subgenerator.